scratch/logging-ns3-script.cc

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+
+#include "ns3/core-module.h"
+
+using namespace ns3;
+
+NS_LOG_COMPONENT_DEFINE ("Logger");
+
+int
+main (int argc, char *argv[])
+{
+
+  Simulator::Run ();
```

Comment:
`Simulator::Run()` and `Simulator::Destroy()` are not needed, I think you can safely remove them!

Reply:
If I remove `Simulator::Run()` and `Simulator::Destroy()` the logs do not output timestamp and node prefix. That is the reason I did this.

Reply:
i would add a comment, then